streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -186,25 +183,9 @@ public void consumerConfigMustContainStreamPartitionAssignorConfig() {
         );
         assertEquals(7L, returnedProps.get(StreamsConfig.WINDOW_STORE_CHANGE_LOG_ADDITIONAL_RETENTION_MS_CONFIG));
         assertEquals("dummy:host", returnedProps.get(StreamsConfig.APPLICATION_SERVER_CONFIG));
-        assertNull(returnedProps.get(StreamsConfig.RETRIES_CONFIG));
-        assertEquals(5, returnedProps.get(StreamsConfig.adminClientPrefix(StreamsConfig.RETRIES_CONFIG)));
         assertEquals(100, returnedProps.get(StreamsConfig.topicPrefix(TopicConfig.SEGMENT_BYTES_CONFIG)));
     }
 
-    @SuppressWarnings("deprecation") // TODO revisit in follow up PR
```

Comment:
Third TODO removed -- as we don't pass `retires` and `retry.backoff.ms` via admin config any longer, this test is not needed any more